src/Microsoft.DotNet.Cli.Utils/AnsiConsole.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Did not update this, left it how it existed. I don't know why this shows as an edit. Looks like it is updated from your PR though. Thanks!